pyccl/halos/profiles.py

```diff
@@ -1551,3 +1554,262 @@ def _Ns(self, M, a):
         M1 = 10.**(self.lM1_0 + self.lM1_p * (a - self.a_pivot))
         alpha = self.alpha_0 + self.alpha_p * (a - self.a_pivot)
         return np.heaviside(M-M0, 1) * (np.fabs(M-M0) / M1)**alpha
+
+
+class SatelliteShearHOD(HaloProfileHOD):
+    def __init__(self, c_M_relation, a1h=0.001, b=-2,
+                 lmax=6, integration_method='FFTLog',
+                 **kwargs):
+        '''
+        Halo HOD class that calculates the satellite galaxy intrinsic shear
+        field in real and fourier space, according to Fortuna et al. 2021.
+        Can be used to compute halo model-based intrinsic alignment
+        (angular) power spectra.
+
+        The satellite intrinsic shear profile in real space is assumed to be
+        .. math::
+            \\gamma^I(r)=a_{1\\mathrm{h}}\\left(\\frac{r}{r_\\mathrm{vir}}
+            \\right)^b \\sin^b\\theta.
+        With :math:`a_{1\\mathrm{h}}` the amplitude of intrinsic alignments on
+        the 1-halo scale, :math:`b` the index defining the radial dependence
+        and :math:`\\theta` the angle defining the projection of the
+        semi-major axis of the galaxy along the line of sight.
+
+        Args:
+            c_M_relation: concentration-mass relation for the halo model.
+            a1h: Amplitude of the satellite intrinsic shear profile.
+            b: Power-law index of the satellite intrinsic shear profile.
+            If zero, the profile is assumed to be constant inside the halo.
```

Comment:
see the syntax for Args here https://github.com/LSSTDESC/CCL/blob/d5408a777ab5fc4b2ae6fada2114a168b9bacee4/pyccl/halos/profiles.py#L645
My understanding is that you need to leave 4 spaces when starting a new line on the same argument. It helps with readability, but it may also be that otherwise rtd won't be able to parse them.